.github/workflows/ci.yml

```diff
@@ -84,6 +84,11 @@ jobs:
         run: echo "TIMEOUT=1200s" >> "$GITHUB_ENV"
       - run: go mod download
         shell: bash
+      - name: go mod tidy
+        shell: bash
```

Comment:
nit I don't think `bash` is really required here, any shell/platform should work

```suggestion
```